manim/_config/main_utils.py

```diff
@@ -449,6 +451,20 @@ def _parse_args_no_subcmd(args: list) -> argparse.Namespace:
         "section of the config file to define the output folder structure",
     )
 
+    # Overrides default false streaming so streaming can happen
+    parser.add_argument(
+        "--livestream",
+        action="store_true",
+        help="Run in streaming mode",
+    )
+
+    # Optional use of IPython for streaming
+    parser.add_argument(
+        "--use-ipython",
+        action="store_true",
+        help="Use IPython for streaming",
+    )
+
```

Comment:
this is entire file doesn't even exist anymore. these options would need to be moved to one of the four files I mentioned for categories under the render command. Argparse has `action="store_true"`, click does `is_flag=True`